File: App_Resources/documentscanner/Android/src/main/java/com/akylas/documentscanner/WorkersContext.java
package com.akylas.documentscanner;

import java.util.Map;
import java.util.concurrent.ConcurrentHashMap;

public final class WorkersContext {
    private final static Map<String, Object> container = new ConcurrentHashMap<String, Object>();

    public static Object getValue(String key) {
        return container.get(key);
    }

    public static void setValue(String key, Object value) {
        if (value != null) {
            container.put(key, value);
        } else {
            container.remove(key);
        }
    }
}

File: App_Resources/documentscanner/Android/src/main/java/com/akylas/documentscanner/WorkersContext.kt
package com.akylas.documentscanner

import java.util.concurrent.ConcurrentHashMap

object WorkersContext {
    private val container: MutableMap<String, Any> = ConcurrentHashMap()
    fun getValue(key: String): Any? {
        return container[key]
    }

    fun setValue(key: String, value: Any?) {
        if (value != null) {
            container[key] = value
        } else {
            container.remove(key)
        }
    }
}